common-service-core/src/main/java/com/ca/mfaas/message/storage/MessageTemplateStorage.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.message.storage;
+
+import com.ca.mfaas.message.core.DuplicateMessageException;
+import com.ca.mfaas.message.template.MessageTemplate;
+import com.ca.mfaas.message.template.MessageTemplates;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Message template storage class
+ */
+public class MessageTemplateStorage {
+    private final Map<String, MessageTemplate> messageTemplateMap = new HashMap<>();
+
+
+    /**
+     * Retrieve message template from the storage by message key
```

Comment:
This one lack something in the end. "Retrieve the message template from the storage by {doing what with} the message key"?